repose-aggregator/components/filters/saml-policy-translation-filter/src/main/scala/org/openrepose/filters/samlpolicy/SamlIdentityClient.scala

```diff
@@ -254,8 +250,7 @@ object SamlIdentityClient {
     (idpId: String) => s"/v2.0/RAX-AUTH/federation/identity-providers/$idpId/mapping"
 
   def buildRetryValue(response: ServiceClientResponse): String = {
-    response.getHeaders
-      .find(hdr => HttpHeaders.RETRY_AFTER.matches(hdr.getName))
+    response.getHeaderElements(HttpHeaders.RETRY_AFTER).asScala.headOption
```

Comment:
This does not map 1:1 with the old behavior. I'm slightly worried that since the `Retry-After` header may contain a comma as part of a date, and not to delimit multiple values, this will mess with stuff. Is there a reason you used `getHeaderElements` rather than `getHeaders`?

Reply:
You would be correct sir. Good catch. This one should have only been `getHeaders()`.